### PR TITLE
Add quotes table to BFX feed

### LINKED ISSUE #61: Urgent - Fix Quote table for Bitfinex Webscoket
## Description

Add quote tables to Bitfinex Websocket

## Checklist

Check the following is included:

- [ ] Reviews, Assignees, Labels, Projects, Milestones
- [ ] Provide response with code snippets if applicable

### PR DESCRIPTION
**Pull request by JuliaPartridge**
## Description

This PR will add the BFX quotes table to the BFX feed.

This closes #61 

## Checklist

- [x] Include a short description of changes to the branch 
- [x] If PR is in reponse to an issue include a keyword with the issue number - e.g. "closes #ISSUENO"
- [x] Include reviews, assignees, labels, projects and milestones
- [x] Code follows style guidelines 
- [x] Code is well commented/documented
- [x] No known bugs

### CODE REVIEW

File: code/processes/tradesbfx.q [hunk 1]
```diff
@@ -33,21 +33,18 @@
...
-//    if[not "hb"~a 1;
-//      $[9=type a 1;
-//        {.bfx.tp(`.u.upd;`bfx_quotes; .bfx.syms[x],y)}[a 0;a 1];
-//        {.bfx.tp(`.u.upd;`bfx_quotes; .bfx.syms[x],y)}[a 0]'[a 1]
-//         ];
-//       ]
-//    ];    
-//  };
+.bfx.quoteupd:{
+  $[99=type a: .j.k x;                                                                          //If message is Dictionary
```
Comment (claire-sloan) [inline code/processes/tradesbfx.q]: whitespace

File: code/processes/tradesbfx.q [hunk 2]
```diff
@@ -33,21 +33,18 @@
...
+      .bfx.msg[a];
```
Comment (claire-sloan) [inline code/processes/tradesbfx.q]: can remove squares

**Comment by jonathonmcmurray** [comment]
Where's the rest of the checklist? Think there's still misunderstanding of the purpose of the templates

File: code/processes/tradesbfx.q [hunk 3]
```diff
@@ -56,10 +53,10 @@
     .lg.o[`SUB;"Subscrption message for trades ",string [x]," sent"];  
   } each .bfx.wildcard syms;                                      
 
-//  {
-//    .bfx.qh .j.j `event`symbol`channel`prec!`subscribe,x,`book ,`R0;
-//    .lg.o[`SUB;"Subscrption message for trades ",string [x]," sent"];
-//  } each .bfx.wildcard syms; 
+  {
+    .bfx.qh .j.j `event`symbol`channel!`subscribe,x,`ticker;
```
Comment (claire-sloan) [inline code/processes/tradesbfx.q]: whitespace here and next line

File: code/processes/tradesbfx.q [hunk 4]
```diff
@@ -33,21 +33,18 @@
...
-//        {.bfx.tp(`.u.upd;`bfx_quotes; .bfx.syms[x],y)}[a 0;a 1];
-//        {.bfx.tp(`.u.upd;`bfx_quotes; .bfx.syms[x],y)}[a 0]'[a 1]
-//         ];
-//       ]
-//    ];    
-//  };
+.bfx.quoteupd:{
+  $[99=type a: .j.k x;                                                                          //If message is Dictionary
+      .bfx.msg[a];
+    if[not 10=type a 1;
```
Comment (claire-sloan) [inline code/processes/tradesbfx.q]: can you add comments here

**Pull request merged**